internal/broker/broker_test.go

```diff
@@ -22,6 +22,10 @@ import (
 
 var defaultIssuerURL string
 
+func pointerTo[T ~string](s T) *T {
+	return &s
+}
```

Comment:
In `authd` itself we use this instead:

```go
func ptrValue[T any](value T) *T {
	return &value
}
```

So for sake of consistency between our projects, please reuse the same.